fbcnms-packages/fbcnms-platform-server/features.js

```diff
@@ -245,6 +245,11 @@ export const arrayConfigs = [
     title: 'Work Order New Status',
     enabledByDefault: false,
   },
+  {
+    id: 'projects_bulk_upload',
+    title: 'Project Bulk Upload',
+    enabledByDefault: true,
```

Comment:
We don't want it to be `enabledByDefault` until we test it internally
`publicAccess` will let you turn on the feature flag yourself on your environment
```suggestion
    enabledByDefault: false,
    publicAccess: true,
```